README.rst

```diff
@@ -0,0 +1,560 @@
+same keystone server than the master region and therefore require their own set
+of credentials. The regions are grouped by *targets*: two regions may be in the
+same *target* if they use the same credential (therefore, their glance servers
+are registered in the same keystone server). The only mandatory *target* is the
+``master`` target, where the master region is. Most of the Glancesync
+configuration, including the criteria to select which images are synchronised,
+is defined at target level. It is okay to create several targets using the same
+credential, for example if some regions only share a minimal set of images and
+others have a broader list.
+
+Glancesync by default does not replace existing images. If an
+image checksum is different between the region to synchronise and the master
+region, a warning is emitted. The user has the option of forcing the
+overwriting of a specific image (optionally renaming the old one) including the
+checksums in a configuration file, using a whilelist or a blacklist.
+
+When the remote image has the same content than the master image, but the
+metadata differs, Glancesync updates the metadata, but only a limited set, to
+avoid overwriting properties considered as local in that glance server. Also
+the system property ``is_public`` is updated.
+
+Glancesync has special support for *AMI* (Amazon Machine Image). Amazon images
+include a reference to a kernel image (*AKI*) and to a ramdisk image (*ARI*),
+but they are named by UUID. Therefore Glancesync has to update this fields to
+reflect the UUIDs in that particular region. 
+
+About UUIDs and image names
+---------------------------
+
+This tool does not synchronise using UUID but names (i.e. an image has the same
+name in all regions, but not the same UUID). Using a UUID to synchronise is
+generally a bad idea, because some problems may arise with the restriction that
+a UUID must be unique. Be aware that it is not possible to replace
+the content of a image, without creating a new one and the old UUID may not be
+reused.  If something similar to an UUID is required, it is better to use a
+metadata field to simulate it.
+
+The downside of using names, is that a region may have more than a image
+with the same name. This is specially challenging, when there is more than a
+image in a destination target, with the name of the image to synchronise. In
+this situation, Glancesync takes the first it found and print a warning with
+the others.
+
+Image names with duplicated names are easy to avoid, with one serious
+exception: when ordinary users can publish their images as public (shared), the
+risk of collision increases and escapes of the control of the user. To avoid
+this, Glancesync allows setting a parameter to ignore images of other tenants.
+Anyway, this is a general problem, not only a synchronisation
+problem, because more than image with the same name is very confusing to the
```

Comment:
because more than image -> due to more that one image with the same name is very confusing to users that want to use them.